src/System.Private.ServiceModel/src/project.lock.json

```diff
@@ -3490,6 +3490,7 @@
       "System.Diagnostics.Contracts >= 4.0.0",
       "System.Diagnostics.Debug >= 4.0.10",
       "System.Diagnostics.Tools >= 4.0.0",
+      "System.Diagnostics.Tracing >= 4.0.20",
```

Comment:
You need to revert this file change and run `msbuild /t:RestorePackages /p:LockDependencies=true`.